docset/windows/dnsserver/Set-DnsServerDiagnostics.md

```diff
@@ -419,7 +419,15 @@ Accept wildcard characters: False
 
 ### -EventLogLevel
 Specifies an event log level.
-Valid values are Warning, Error, and None.
+Valid values are:
+
+0: No Events
+
+1: Errors Only
+
+2: Errors and warnings
+
+7: All Events
```

Comment:
```suggestion
- 3-7: All Events
```

Reply:
This and if you can apply the same update for 2012 and 2012 R2 versions are more than enough for me. Thanks!

Reply:
This update and apply the same for 2012 and 2012 R2 versions are more than enough for me. Thanks!